samples/newChat/js/message.js

```diff
@@ -0,0 +1,394 @@
+'use strict';
+
+function Message() {
+    this.container = null;
+    this.attachmentPreviewContainer = null;
+    this.limit = appConfig.messagesPerRequest || 50;
+
+    this.dialogTitle = null;
+    this._typingTimer = null;
+    this._typingTime = null;
+    this.typingUsers = {};
+}
+
+Message.prototype.init = function(){
+    var self = this;
+
+    self.container = document.querySelector('.j-messages');
+    self.attachmentPreviewContainer = document.querySelector('.j-attachments_preview');
+    self.dialogTitle = document.querySelector('.j-content__title');
+
+    document.forms.send_message.addEventListener('submit', function(e){
+        e.preventDefault();
```

Comment:
preventDefault doesn't stop further propagation of the event through the DOM. event.stopPropagation should be used for that.